pkg/logger/progress_logger_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logger_test
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	regv1 "github.com/google/go-containerregistry/pkg/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/cli-runtime"
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/logger"
+)
+
+func TestProgressLogger(t *testing.T) {
+	t.Run("Test progress logger", func(t *testing.T) {
+		outputwriter := &bytes.Buffer{}
+		scheme := runtime.NewScheme()
+		c := cli.NewDefaultConfig("test", scheme)
+		c.Stdout = outputwriter
+		levelLogger := logger.NewSinkLogger("", int32Ptr(1), nil)
+		testprogresslogger := logger.NewNoopProgressBar(levelLogger)
+		processData(testprogresslogger)
```

Comment:
why is the func created for this? 
Consider gofunc to create and pass info to channel.